frontend/src/pages/logout/Logout.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect } from "react";
+
+const Logout: React.FC = () => {
+    useEffect(() => {
```

Comment:
@egdagger do we need the use effect or just 
`window.location.href = "/logout";`

Reply:
With the change of location works fine. Already removed the use effect